.github/workflows/checks.yml

```diff
@@ -202,14 +202,12 @@ jobs:
     strategy:
       fail-fast: false
     env:
-      GOERLI_INTEGRATION_RPC_URL: ${{ secrets.INTEGRATION_RPC_URL }}
-      GOERLI_TESTNET_RPC_URL: ${{ secrets.TESTNET_RPC_URL }}
       SEPOLIA_INTEGRATION_RPC_URL: ${{ secrets.SEPOLIA_INTEGRATION_RPC_URL }}
       SEPOLIA_TESTNET_RPC_URL: ${{ secrets.SEPOLIA_TESTNET_RPC_URL }}
```

Comment:
Let's make sure we use our new rpc urls, if I can see correctly at the moment we are not (they were updated some time ago, whereas new nodes were made yesterday)